codec.proto

```diff
@@ -40,11 +40,15 @@ message EventData {
   EventBlock block = 2;
 }
 
+message TransactionData {
+  TxResult   tx    = 1;
+  EventBlock block = 2;
+}
+
```

Comment:
I'm fairly sure you set this file up to be alphabetized before. This should be further down the file if so